File: LeetCode/Easy/1678-goal-parser-interpretation/try-4.py
/*
 * Problem #1678: Goal Parser Interpretation
 * Difficulty: Easy
 * Submission: Try 4
 * status: Accepted
 * Language: python3
 * Date: 7/9/2026, 10:33:53 AM
 * Link: https://leetcode.com/problems/goal-parser-interpretation/
 */

class Solution:
    def interpret(self, command: str) -> str:
        return command.replace("()", "o").replace("(al)", "al")

File: LeetCode/Easy/367-valid-perfect-square/try-4.c
/*
 * Problem #367: Valid Perfect Square
 * Difficulty: Easy
 * Submission: Try 4
 * status: Accepted
 * Language: c
 * Date: 7/10/2026, 11:18:04 AM
 * Link: https://leetcode.com/problems/valid-perfect-square/
 */

bool isPerfectSquare(int num) {
    double ans=sqrt(num);
    if(ans!=floor(ans)){
        return false;
    }
    return true;
}
